### PR TITLE
Feature/fix install keys script

### PR DESCRIPTION
**Pull request by tleyden**
TODO: run against vagrant

#### Fixes #.

- [ ] Ran `flake8`

#### Changes proposed in this pull request:

- 
-

### CODE REVIEW

**Comment by tleyden** [comment]
Fixes #885

File: libraries/provision/ansible/playbooks/files/gateload_config.json [hunk 1]
```diff
@@ -20,6 +20,5 @@
     "AuthType": "basic",
     "Password": "password",
     "StatsdEnabled":true,
-    "FeedType": "{{ feed_type }}",
```
Comment (sethrosetter) [inline libraries/provision/ansible/playbooks/files/gateload_config.json]: Why is this removed?

**Comment by tleyden** [comment]
Moved to https://github.com/couchbaselabs/mobile-testkit/pull/1008
